### PR TITLE
feat: added 'my account' view

### PR DESCRIPTION
**Pull request by pswiderski123**

### CODE REVIEW

File: src/app/my-account/my-account/my-account.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  styleUrls: ['./my-account.component.scss'],
+})
+export class MyAccountComponent implements OnInit {
+  public myAnnouncements: Array<OffersBaseOffer> = [];
+  public myAccountPersonalData: MyAccountPersonalData | null | undefined;
+
+  constructor(private myAccountService: MyAccountService) {}
+
+  public ngOnInit() {
+    this.myAccountService.getPersonalData().subscribe((data: MyAccountPersonalData) => {
```
Comment (broslukasz) [inline src/app/my-account/my-account/my-account.component.ts:18]: `public myAccountPersonalData: Observable<MyAccountPersonalData>`

`this.myAccountPersonalData$ = this.myAccountService.getPersonalData()`

in html
`[myAccountPersonalData]="myAccountPersonalData | async"`

File: src/app/shared/models/my-account-path.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const MyAccountPath: string = 'moje-konto';
```
Comment (broslukasz) [inline src/app/shared/models/my-account-path.ts:1]: Suggested refactor of a file `help-type-path.enum` to `core-path.enum`

with changes:

```
export enum CorePath {
  FindHelp = 'znajdz-pomoc',
  GiveHelp = 'dodaj-ogloszenie',
  MyAccount = 'moje-konto'
}
```

File: src/app/my-account/my-account/my-account.component.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <header>
+    <h2>{{ "HELPUA" | translate }}</h2>
+    <h1>{{ "SEARCH_FOR_HELP" | translate }}</h1>
+  </header>
+  <section>
+    <p class="text">Ustawienia konta</p>
+    <app-my-account-settings [myAccountPersonalData]="myAccountPersonalData"></app-my-account-settings>
+  </section>
+  <button color="primary" mat-flat-button>+ DODAJ OGŁOSZENIE</button>
```
[the remark this answers is not in the document]
Comment (pswiderski123) [reply src/app/my-account/my-account/my-account.component.html:10]: done

File: src/app/shared/models/my-account-path.ts (hunk 2 above)
Comment (pswiderski123) [reply src/app/shared/models/my-account-path.ts:1]: done

File: src/app/my-account/my-account-settings/my-account-settings.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
```
[the remark this answers is not in the document]
Comment (pswiderski123) [reply src/app/my-account/my-account-settings/my-account-settings.component.spec.ts:1]: done

File: src/app/my-account/my-account/my-account.component.ts (hunk 1 above)
Comment (pswiderski123) [reply src/app/my-account/my-account/my-account.component.ts:18]: done

File: src/app/my-account/my-account-settings/my-account-settings.component.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<mat-card *ngIf="myAccountPersonalData" class="container">
+  <div class="row">
+    <div class="col-sm-12 col-md-6">
+      <p class="font-small">Imię</p>
+      <p class="font-large pb-sm-2 pb-md-0">{{myAccountPersonalData.name }}</p>
+      <p>Twoje imię pobieramy z profilu zaufanego.</p>
```
[the remark this answers is not in the document]
Comment (pswiderski123) [reply src/app/my-account/my-account-settings/my-account-settings.component.html]: done

File: src/app/core/site-header/site-header.component.ts [hunk 6]
```diff
@@ -50,4 +50,8 @@ export class SiteHeaderComponent {
   useLanguage(langCode: string) {
     this.translateService.use(langCode);
   }
+
+  public navigateToMyAccount(): void {
+    this.router.navigate(['moje-konto']);
```
Comment (broslukasz) [inline src/app/core/site-header/site-header.component.ts:55]: CorePath.MyAccount

File: src/app/core/site-header/site-header.component.html [hunk 7]
```diff
@@ -66,6 +66,10 @@
           <li class="navbar-item navbar-big mobile-only">
             <a class="navbar-link" href="#">{{ "ANNOUNCEMENTS" | translate }}</a>
           </li>
+          <li (click)="navigateToMyAccount()" class="navbar-item button">
+            <mat-icon class="button-icon" fontSet="material-icons-outlined">perm_identity</mat-icon>
+            <span class="button-title">Moje konto</span>
```
Comment (broslukasz) [inline src/app/core/site-header/site-header.component.html:71]: missing translation

File: src/app/shared/models/core-path.enum.ts [hunk 8]
```diff
@@ -0,0 +1,5 @@
+export enum CorePath {
+  Find = 'szukaj-pomocy',
```
Comment (broslukasz) [inline src/app/shared/models/core-path.enum.ts:2]: suggestion: to have `GiveHelp`m `FindHelp`

File: src/app/shared/models/core-path.enum.ts [hunk 9]
```diff
@@ -0,0 +1,5 @@
...
+  Give = 'daj-pomoc',
+  MyAccount = 'moje-konto',
```
Comment (broslukasz) [inline src/app/shared/models/core-path.enum.ts:4]: Please rebase, those paths have changed

File: src/app/core/site-header/site-header.component.ts [hunk 10]
```diff
@@ -50,4 +52,8 @@ export class SiteHeaderComponent {
   useLanguage(langCode: string) {
     this.translateService.use(langCode);
   }
+
+  public navigateToMyAccount(): void {
+    this.router.navigate([this.corePath.MyAccount]);
```
Comment (broslukasz) [inline src/app/core/site-header/site-header.component.ts:57]: wystarczy CorePath.MyAccount

bez przypisywania do zmiennej

File: src/app/shared/models/core-path.enum.ts [hunk 11]
```diff
@@ -0,0 +1,5 @@
+export enum CorePath {
```
Comment (broslukasz) [inline src/app/shared/models/core-path.enum.ts:1]: Zmieniły się ściezki, proszę uwzględnij
https://github.com/coi-gov-pl/gui-web-pomogamukrainie-forum/blob/cbb736c6ed9037e5a745380d6d657ad35f0f6f01/src/app/shared/models/help-type-path.enum.ts

File: src/api/model/my-account-personal-data.ts [hunk 12]
```diff
@@ -0,0 +1,4 @@
+export interface MyAccountPersonalData {
```
Comment (broslukasz) [inline src/api/model/my-account-personal-data.ts:1]: Please squash commits into one if github does not provide it automatically when merge

**Review by broslukasz — APPROVED** [review]

File: src/app/my-account/my-account-settings/my-account-settings.component.ts [hunk 13]
```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { MyAccountPersonalData } from '../../../api/model/my-account-personal-data';
+
+@Component({
+  selector: 'app-my-account-settings',
+  templateUrl: './my-account-settings.component.html',
+  styleUrls: ['./my-account-settings.component.scss'],
+})
+export class MyAccountSettingsComponent {
+  @Input() public myAccountPersonalData: MyAccountPersonalData | null | undefined;
```
Comment (broslukasz) [inline src/app/my-account/my-account-settings/my-account-settings.component.ts:10]: prawdopodobnie wystarczy `myAccountPersonalData: MyAccountPersonalData`

**Pull request merged**